### PR TITLE
New package: incredimo.textra version v2024.09.18.164358

### PR DESCRIPTION
**Pull request by incredimo**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/174144)

### CODE REVIEW

**Comment by incredimo** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ADO Build not found.

(Automated response - build 895.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\incredimo.textra__DefaultSource\textra.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\textra.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 896.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-09-19 09:35:06.747 [FAIL] Installer failed security check. Url: https://github.com/u-tra/textra/releases/download/v
> 2024.09.18.164358/textra.exe Result: 0x80004005

(Automated response - build 896.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\incredimo.textra__DefaultSource\textra.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\textra.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 896.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-09-20 18:51:28.856 [FAIL] Installer failed security check. Url: https://github.com/u-tra/textra/releases/download/v
> 2024.09.18.164358/textra.exe Result: 0x80004005

(Automated response - build 896.)

File: manifests/i/incredimo/textra/v2024.09.18.164358/incredimo.textra.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: incredimo.textra
+PackageVersion: v2024.09.18.164358
+InstallerType: portable
+Commands:
+- textra
+Installers:
```
Comment (stephengillie) [inline manifests/i/incredimo/textra/v2024.09.18.164358/incredimo.textra.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Automatic Validation ended with:
> 
> > Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\incredimo.textra__DefaultSource\textra.exe returned exit code: -1073741515
> > Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\textra.exe returned exit code: -1073741515
> 
> Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).
> 
> (Automated response - build 896.)

Manual Validation ended with: 
image

**Pull request closed without merging**